sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosRowConverter.scala

```diff
@@ -257,6 +269,8 @@ private object CosmosRowConverter
               parseOperationType(objectNode)
             case StructField(CosmosTableSchemaInferrer.TtlExpiredAttributeName, BooleanType, _, _) =>
               parseTtlExpired(objectNode)
+            case StructField(CosmosTableSchemaInferrer.LsnAttributeName, LongType, _, _) =>
```

Comment:
Should we map to `LongType` or is the attribute mapping enough?

Reply:
Sorry - don't understand the question? DO you mean whether we should attempt the parsing even if _lsn is provided in user schema but of different data type?

Reply:
I meant as:
`case StructField(CosmosTableSchemaInferrer.LsnAttributeName, _, _, _) =>`

Reply:
Json -> Schema of the _lsn could be an Integer or Long depending on the value and how json interprets it.

Reply:
Our _lsn values could exceed 2147483647 - http://spark.apache.org/docs/latest/sql-ref-datatypes.html

I don't quite see why we shouldn't force folks to use the "correct" data type for LSN?